classes/user/Collector.inc.php

```diff
@@ -33,8 +39,7 @@ class Collector implements CollectorInterface
     public const STATUS_DISABLED = 'disabled';
     public const STATUS_ALL = null;
 
-    /** @var DAO */
-    public $dao;
+    public DAO $dao;
```

Comment:
I've just noticed that we're using PHP 7.4 on `/main`, so I'll make more updates here.